jest.config.cjs

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  coverageDirectory: '<rootDir>/test-results/coverage',
+  collectCoverageFrom: ['src/**/*.{ts,tsx}', '!src/**/*-stories.tsx', '!src/**/index.ts'],
+  coverageReporters: ['lcovonly', 'text'],
+  moduleFileExtensions: ['ts', 'tsx', 'js'],
+  moduleNameMapper: {
+    '^.+\\.(md|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/config/jest/FileStub.js',
+    '^.+\\.css$': '<rootDir>/config/jest/CSSStub.js',
```

Comment:
I think we are missing those two stub files

Reply:
Oh, we can probably remove those... we don't import any from the components